editor.planx.uk/src/components/Header/Header.tsx

```diff
@@ -231,58 +233,70 @@ const Breadcrumbs: React.FC = () => {
     state.previewEnvironment === "standalone",
   ]);
 
+  const flowStatus = useStore((state) => state.status) as
```

Comment:
```suggestion
  const flowStatus = useStore((state) => state.flowStatus)
```

This is saved in the store as `flowStatus` not `status` 👍

Reply:
Please see comment below - fixed in [daa2b7a](https://github.com/theopensystemslab/planx-new/pull/4497/commits/daa2b7aeca44f1e0a3837df445679a0c28440134)